src/views/Instance/Instance.js

```diff
@@ -3,35 +3,28 @@
 
 import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { CytoViz, HierarchicalComboBox } from '@cosmotech/ui';
-import { sortScenarioList } from '../../utils/SortScenarioListUtils';
+import { CytoViz } from '@cosmotech/ui';
 import { parseError } from '../../utils/ErrorsUtils';
 import { STATUSES } from '../../state/commons/Constants';
 import { AppInsights } from '../../services/AppInsights';
 import { fetchData, processGraphElements } from './data';
 import useStyles from './style';
+import { CurrentScenarioSelector } from '../../components';
 import { useTheme } from '@mui/styles';
 import { useInstance } from './InstanceHook';
+import { useRedirectionToScenario } from '../../hooks/RouterHooks';
```

Comment:
This hook has been removed from the `InstanceHook` file to be imported here, was it on purpose ?

Reply:
done